integration-tests/run-test.sh

```diff
@@ -125,8 +125,8 @@ check_env_vars "$@" # Pass all args for consistency, though check_env_vars doesn
 parse_options "$@" # Parses options and sets CLEANUP, NO_CVE
 
 decrypt_secrets
-delete_old_branches "${component_repo_name}" "${component_type}" 2
-create_github_branch
```

Comment:
can't this function be deleted then? same for delete_old_branches

Reply:
yes. I can do that.